src/site/content/en/blog/badging-api/index.md

```diff
@@ -86,67 +85,63 @@ Examples of sites that may use this API include:
-  implementation of badging on the platform.
-{% endAside %}
+The Badging API works on Windows, and macOS. Support for Chrome OS is in
+development and will be available in a future release of Chrome.
+Unfortunately, Android is not supported because it requires you to show a
+notification, though this may change in the future.
 
 ### Register for the origin trial {: #ot }
 
+{% Aside 'caution' %}
+In Chrome 79 (stable Dec 2019), the first origin trial ended, and was replaced
+by a new origin trial that uses a slightly different API design. Unfortunately,
+it is not backwards compatible, and calls to the previous API will fail. See
+[Supporting both the first and second origin trial](#backwards-compat) for
+details on how to provide backwards compatibility.
+{% endAside %}
+
 {% include 'content/origin-trials.njk' %}
 
 {% include 'content/origin-trial-register.njk' %}
 
+If necessary, you can
+[request a token](https://developers.chrome.com/origintrials/#/view_trial/1711367858400788481)
+for Chrome 78 and earlier.
+
 ### Alternatives to the origin trial
 
 If you want to experiment with the Badging API locally, without an origin trial,
 enable the `#enable-experimental-web-platform-features` flag in `chrome://flags`.
 
-### Using the Badging API during the origin trial
-
-{% Aside 'gotchas' %}
-  During the origin trial, the API will be available via
-  `window.ExperimentalBadge`. The code below is based on the current design,
-  and will change before it lands in the browser as a standardized API.
-{% endAside %}
+### Setting and clearing a badge
 
 To use the Badging API, your web app needs to meet
 [Chrome's installability criteria](https://developers.google.com/web/fundamentals/app-install-banners/#criteria),
 and users must add it to their home screens.
 
-The `ExperimentalBadge` interface is a member object on `window`. It contains
-two methods:
+The Badge API consists of the following methods on `navigator`:
 
-* `set([number])`: Sets the app's badge. If a value is provided, set the badge
-  to the provided value otherwise, display a plain white dot (or other flag as
-  appropriate to the platform).
-* `clear()`: Removes app's badge.
+* `setExperimentalAppBadge([number])`: Sets the app's badge. If a value is
+  provided, set the badge to the provided value otherwise, display a plain
+  white dot (or other flag as appropriate to the platform).
+* `clearExperimentalAppBadge()`: Removes app's badge.
 
 For example:
 
 ```js
-// In a web page
+// Set the badge
 const unreadCount = 24;
-window.ExperimentalBadge.set(unreadCount);
+navigator.setExperimentalAppBadge(unreadCount);
+
+// Clear the badge
+navigator.clearExperimentalAppBadge();
 ```
 
-`ExperimentalBadge.set()` and `ExperimentalBadge.clear()` can be called from
+`setExperimentalAppBadge()` and `clearExperimentalAppBadge()` can be called from
 a foreground page, or potentially in the future, a service worker. In either
```

Comment:
Unless we're really sure we going to do this, I wouldn't say what we want to do in the future.

Reply:
There's some debate about how it will work from push but we're pretty confident that we're going to expose it from the service worked in the near future.